src/Uno.Extensions.Templates/content/unoapp-extensions/Directory.Packages.props

```diff
@@ -22,26 +22,32 @@
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageVersion Include="NUnit" Version="3.13.2" />
     <PackageVersion Include="NUnit3TestAdapter" Version="4.0.0" />
+    <!--#if (server && use-serilog)-->
```

Comment:
Do we really need all these conditional flags in the Packages.props - all this is doing is setting version number for when packages are used in cpm? I get that this keeps these files clean but I'm just thinking of the overhead of maintaining more conditional flags